packager/packager.cc

```diff
@@ -614,7 +614,8 @@ Status CreateTextJobs(
         }
```

Comment:
Line 605 should be updated too:

```c++
if (hls_listener && !stream.dash_only)
```

Reply:
Sorry guys for not responding previously, I was distracted by other things but I started working on this also.
I would suggest adding also at line 610:
```C
if ((stream.segment_template.empty() || !stream.output.empty()) && !stream.dash_only)
```

Reply:
Sorry guys for not responding previously, I was distracted by other things but I started working on this also. Thank you @sr1990 for your fix.
I would suggest adding also at line 610:
```C
if ((stream.segment_template.empty() || !stream.output.empty()) && !stream.dash_only)
```

Reply:
@popraduvmp No worries! The code at line 610 is a sub-condition under the `if` at line 605, and `stream.dash_only` is already guaranteed to be false, so the additional check is unnecessary.

Reply:
@kqyang You are so right... My bad, still early for me, I just copy pasted my changes quickly and didn't check.